scripts/count-median.py

```diff
@@ -91,7 +91,10 @@ def main():
     ksize = htable.ksize()
 
     print('writing to', output_filename, file=sys.stderr)
-    output = open(output_filename, 'w')
+    if output_filename in ('-', '/dev/stdin'):
+       output = sys.stdout
+    else:
+       output = open(output_filename, 'w')
```

Comment:
I disagree. It is a nice default but one should be allowed to stream in and write to a file. Especially bad as we say to the user in line 93 that we are writing to the file name they specified but we immediately go and do something different.